difference_of_squares.rb

```diff
@@ -0,0 +1,29 @@
+class Squares
+  def initialize(nutual_num)
+    @nutual_num=nutual_num
+    square_of_sum
+    sum_of_squares
+    difference
+  end
+
+  def square_of_sum
+    sum = 0
+    (1..@nutual_num).each { |num| sum += num }
+    sum ** 2
+  end
+
+  def sum_of_squares
+    squares = 0
+    (1..@nutual_num).each { |num| squares += (num ** 2) }
+    squares
+  end
+
+  def difference
```

Comment:
@keyurgohil2426 Both the methods `square_of_sum` and `sum_of_squares` will return something then no need to further define `diffrence_of = 0`
```
def difference
   square_of_sum - sum_of_squares
end
```